components/status_dropdown/status_dropdown.tsx

```diff
@@ -266,16 +276,16 @@ export default class StatusDropdown extends React.PureComponent {
         );
     }
 
-    refCallback = (ref) => {
+    refCallback = (ref: MenuWrapper): void => {
         if (ref) {
             this.setState({
-                width: ref.state.width,
```

Comment:
Hi @devinbinnie , there is no state width in MenuWrapper, should I update state width in this component ?

Reply:
@arvinDarmawan I think this logic is leftover from a previous version of `MenuWrapper`. And since we don't use either of those variables anywhere, I think we can actually remove the whole ref here since it doesn't do anything.

Reply:
ok, done @devinbinnie